tsconfig.json

```diff
@@ -1,8 +1,25 @@
 {
+  "extends": "./tsconfig.build.json",
   "compilerOptions": {
-    "target": "es2018",
-    "module": "commonjs",
-    "esModuleInterop": true,
-    "lib": ["esnext"]
-  }
+    "baseUrl": "./src/packages",
+    "paths": {
+      "@prisma/client": ["client/src/runtime"],
+      "@prisma/sdk": ["sdk/src"],
+      "@prisma/migrate": ["migrate/src"],
+      "@prisma/generator-helper": ["generator-helper/src"],
+      "@prisma/engine-core": ["engine-core/src"],
+      "@prisma/debug": ["debug/src"],
+      "@prisma/cli": ["cli/src"],
+      "@prisma/get-platform": ["../../../engines-wrapper/packages/get-platform/src"],
```

Comment:
That requires us to clone the engines-wrapper repo next to prisma repo on our machine with that exact name to work then?

Reply:
Yes, but if you don't it will fall back on dist types.